src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -707,6 +707,26 @@ private static FlowAnalysisAnnotations DecodeFlowAnalysisAttributes(PEModule mod
             return annotations;
         }
 
+        internal override ImmutableHashSet<string> NotNullIfParameterNotNull
+        {
+            get
+            {
+                // TODO2 cache?
+                var attributes = GetAttributes();
+                ImmutableHashSet<string>.Builder set = null;
+                foreach (var attribute in attributes)
+                {
+                    if (attribute.IsTargetAttribute(this, AttributeDescription.NotNullIfNotNullAttribute))
+                    {
+                        set ??= ImmutableHashSet.CreateBuilder<string>();
+                        set.Add(attribute.DecodeNotNullIfNotNullAttribute());
```

Comment:
>DecodeNotNullIfNotNullAttribute() [](start = 42, length = 33)

It looks like we're adding, even if `DecodeNotNullIfNotNullAttribute` returns null.